app/presenters/arclight/show_presenter.rb

```diff
@@ -5,6 +5,7 @@ module Arclight
   class ShowPresenter < Blacklight::ShowPresenter
     def heading
       title = super
+      title = nil if title.blank? || title == @document.id.to_s
```

Comment:
Also, can we use `document` instead of `@document` to be more consistent with the other usage in this method?